tests/dags/common/popularity/test_sql.py

```diff
@@ -500,3 +503,25 @@ def test_image_view_calculates_std_pop(postgres_with_image_table, table_info):
             rd["fid_d"] == 0.75,
         ]
     )
+
+
+@pytest.mark.parametrize(
+    "ddl_filename, metrics",
+    [
+        ("0004_openledger_image_view.sql", sql.IMAGE_POPULARITY_METRICS),
+        ("0007_openledger_audio_view.sql", sql.AUDIO_POPULARITY_METRICS),
+    ],
+)
+def test_ddl_matches_definitions(ddl_filename, metrics):
```

Comment:
So cool!

Reply:
Awesome!

Reply:
Thanks! I always think of adding it to the SQL file but I didn't even remember the logic file existed 😅